src/App.js

```diff
@@ -0,0 +1,61 @@
+import React,{useState } from "react";
+import { Button } from './components/Button/Button';
+
+export default function App() {
+  const [users, setUsers] = useState();
+  const [error, setError] = useState();
+
+  const handleClick = async () => {
+   
+    try {
+      const response = await fetch(
+        "https://jsonplaceholder.typicode.com/users"
+      );
+      const data = await response.json();
+        
+      getAllAvatar(data);
+    } catch (error) {
+        setError("We couldn't make the request to get the user");
+    }
+  }
+
+    const getAllAvatar = async (data) => {
+      let newUsersList = [];
+      try {
+        for (let i = 0; i < data.length; i++) {
+          const response = await fetch(
+            "https://avatars.dicebear.com/api/male/${data[i].username}.svg?background=%230000ff"
```

Comment:
`"https://avatars.dicebear.com/api/male/${data[i].username}.svg?background=%230000ff"`

That doesn't work, is incorrectly written. 

must be: (stop copy and pasting)
```suggestion
            `https://avatars.dicebear.com/api/male/${data[i].username}.svg?background=%230000ff`
```